acq4/devices/PatchPipette/states/_base.py

```diff
@@ -306,25 +324,67 @@ def depthBelowSurface(self, pos=None):
     def aboveSurface(self, pos=None):
         return self.depthBelowSurface(pos) < 0
 
+    def maybeVisuallyTrackTarget(self):
```

Comment:
Idly wondering if there are ways to prevent the base class becoming more complex in order to support features only used by a few states.